forte/evaluation/base/base_evaluator.py

```diff
@@ -47,3 +47,10 @@ def get_result(self) -> Any:
         r"""The evaluator gather the results and the score can be obtained here.
         """
         raise NotImplementedError
+
+    def record(self, record_meta: Dict):
+        pass
+
+    @classmethod
+    def expected_type(cls) -> Dict:
```

Comment:
Could you write the docstring here so the implementer of this class knows what to implement here?